tox.ini

```diff
@@ -1,152 +1,62 @@
 skip_install = true
-# N.B. "hypothesis" in the list below is the property-based testing library,
-#      not our own code.
-deps =
-    coverage
-    mock
-    pytest
-    hypothesis
-    factory-boy
-    -rrequirements.txt
-passenv =
-    TEST_DATABASE_URL
-    ELASTICSEARCH_URL
-    PYTEST_ADDOPTS
-commands =
-    coverage run --parallel --source h,tests/h -m pytest -Werror {posargs:tests/h/}
-
-[dev]
-deps = -rrequirements-dev.in
-passenv =
-    ALLOWED_ORIGINS
-    AUTHORITY
-    BOUNCER_URL
-    CLIENT_OAUTH_ID
-    CLIENT_RPC_ALLOWED_ORIGINS
-    CLIENT_URL
-    CONFIG_URI
-    MODEL_CREATE_ALL
-    SEARCH_AUTOCONFIG
-    SENTRY_DSN
-    USE_HTTPS
-    WEBSOCKET_URL
-whitelist_externals = sh
-commands =
-    sh bin/hypothesis --dev init
-    {posargs:sh bin/hypothesis devserver}
-
-[testenv:py27-dev]
-deps = {[dev]deps}
-passenv = {[dev]passenv}
-whitelist_externals = {[dev]whitelist_externals}
-commands = {[dev]commands}
-
-[testenv:py36-dev]
-deps = {[dev]deps}
-passenv = {[dev]passenv}
-whitelist_externals = {[dev]whitelist_externals}
-commands = {[dev]commands}
-
-[testenv:py37-dev]
-deps = {[dev]deps}
-passenv = {[dev]passenv}
-whitelist_externals = {[dev]whitelist_externals}
-commands = {[dev]commands}
-
-[functional]
-deps =
-    pytest
-    webtest
-    factory-boy
-    -rrequirements.txt
 passenv =
-    BROKER_URL
-    ELASTICSEARCH_URL
-    TEST_DATABASE_URL
-    PYTEST_ADDOPTS
-commands = pytest -Werror {posargs:tests/functional/}
-
-[testenv:functional]
-deps = {[functional]deps}
-passenv = {[functional]passenv}
-commands = {[functional]commands}
-
-[testenv:functional-py36]
-deps = {[functional]deps}
-passenv = {[functional]passenv}
-commands = {[functional]commands}
-
-[testenv:clean]
-deps = coverage
-skip_install = true
-commands = coverage erase
-
-[testenv:coverage]
-deps = coverage
-skip_install = true
-commands =
-    coverage combine
-    coverage report
-
-[testenv:codecov]
-deps = codecov
-skip_install = true
-passenv = CI TRAVIS*
-commands = codecov
-
-[docs]
-changedir = docs
-deps =
-    sphinx
-    sphinx_rtd_theme
-    sphinx-autobuild
-
-[testenv:docs]
-changedir = {[docs]changedir}
-deps = {[docs]deps}
-commands = sphinx-autobuild -BqT -b dirhtml -d {envtmpdir}/doctrees . {envtmpdir}/html
-
-[testenv:checkdocs]
-changedir = {[docs]changedir}
-deps = {[docs]deps}
-commands = sphinx-build -qTWn -b dirhtml -d {envtmpdir}/doctrees . {envtmpdir}/html
-
-[docstrings]
-commands = sphinx-apidoc -ePMF -a -H "Dooccsstrinngs!!" --ext-intersphinx --ext-todo --ext-viewcode -o {envtmpdir}/rst .
-deps =
-    {[docs]deps}
-    {[testenv]deps}
-
-[testenv:docstrings]
-deps = {[docstrings]deps}
-commands =
-    {[docstrings]commands}
-    sphinx-autobuild -BqT -z h -z tests -b dirhtml {envtmpdir}/rst {envtmpdir}/dirhtml
-
-[testenv:checkdocstrings]
-deps = {[docstrings]deps}
-commands =
-    {[docstrings]commands}
-    sphinx-build -qTn -b dirhtml {envtmpdir}/rst {envtmpdir}/dirhtml
-
-[testenv:lint]
+    dev: ALLOWED_ORIGINS
```

Comment:
`ALLOWED_ORIGINS` was removed in c7441a638eb0c979ef585c7a16c299d6e88b00e4

Reply:
These are already in the `tox.ini` file on master I think. I'll send a separate PR to remove them